examples/auto-ads.amp.html

```diff
@@ -3,9 +3,10 @@
   <head>
     <meta charset="utf-8">
     <title>AMP #0</title>
-    <link rel="canonical" href="amps.html" >
+    <link rel="canonical" href="http://amp.autoplaced.com" >
     <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
     <script async custom-element="amp-auto-ads" src="https://cdn.ampproject.org/v0/amp-auto-ads-0.1.js"></script>
+    <script async custom-element="amp-sticky-ad" src="https://cdn.ampproject.org/v0/amp-sticky-ad-1.0.js"></script>
```

Comment:
Yes. page without `amp-ad` script is valid because there's a refactor to move ad related code from builtin to extensions folder. However validator will throw warning against it. 

> <script async custom-element="amp-ad" src="https://cdn.ampproject.org/v0/amp-ad-0.1.js"></script> 
> Note: amp-ad may still work without this script, but we highly recommend it for future compatibility